pkg/registry/digest/digest_test.go

```diff
@@ -0,0 +1,64 @@
+package digest
+
+import (
+	"github.com/docker/docker/api/types"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	log "github.com/sirupsen/logrus"
+	"os"
+	"testing"
+)
+
+func TestDigest(t *testing.T) {
+	log.SetLevel(log.DebugLevel)
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Digest Suite")
+}
+
+var image = types.ImageInspect{
+	ID: "sha256:6972c414f322dfa40324df3c503d4b217ccdec6d576e408ed10437f508f4181b",
+	RepoTags: []string {
+		"ghcr.io/k6io/operator:latest",
+	},
+	RepoDigests: []string {
+		"ghcr.io/k6io/operator@sha256:d68e1e532088964195ad3a0a71526bc2f11a78de0def85629beb75e2265f0547",
+	},
+}
+
+var (
+	DH_USERNAME = os.Getenv("CI_INTEGRATION_TEST_REGISTRY_DH_USERNAME")
+	DH_PASSWORD = os.Getenv("CI_INTEGRATION_TEST_REGISTRY_DH_PASSWORD")
+	GH_USERNAME = os.Getenv("CI_INTEGRATION_TEST_REGISTRY_DH_USERNAME")
+	GH_PASSWORD = os.Getenv("CI_INTEGRATION_TEST_REGISTRY_DH_PASSWORD")
```

Comment:
```suggestion
	GH_USERNAME = os.Getenv("CI_INTEGRATION_TEST_REGISTRY_GH_USERNAME")
	GH_PASSWORD = os.Getenv("CI_INTEGRATION_TEST_REGISTRY_GH_PASSWORD")
```

Reply:
Some refactoring inc. tomorrow. That change is in there, sort of! 😅

Reply:
Yeah, no worries. My typo alarm just went off 😁
Felt like a typical problem one could spend way too long trying to figure out why the tests fail for no good reason.